### DISCUSSION TITLE
Sprite addition

### DISCUSSION THREAD

**Discussion by Comicrocket** [mriscoc/Ender3V2S1#221] [Q&A]
Could you have it so that we can install a creality sprite extruder on a V2?

**Comment by Verisper** [comment]
I swapped to a sprite on my E3v2 and used the suggested step change from the sprite reference book thing then installed the bltouch version of the firmware for my board. I had to do some calibration to get everything squared away since the sprite is a bit bulkier in areas. I haven't had any issues as far as I can tell. Granted I'm still relatively new to the sport
